backend/endpoints/files.py

```diff
@@ -0,0 +1,98 @@
+from fastapi import  HTTPException, status
+from schema.response import ResponseModel
+from starlette.responses import JSONResponse, Response
+from fastapi import FastAPI, UploadFile, File, APIRouter
+from fastapi.responses import FileResponse, JSONResponse
+from utils.message_utils import get_message, get_room_messages
+import os
+import zipfile
+import io
+from zipfile import ZipFile
+router = APIRouter()
+
+"""
+ A get method endpoint to Handle retrieving and Viewing of uploaded files in a DM
+
+"""
+
+# {
+#     "organization_id" :"6373eb474746182adae97314",
+#     "room_id" : "6373eb4f4746182adae97316"
+
+        #         org_id:
+        # 6373eb474746182adae97314
+        # room_id:
+        # 6373eb4f4746182adae97316
+
+        #  Use:
+        # org_id: 619ba4
+        # room_id: 123456
+# }
+
+
+def zipfiles(filenames: str):
+    zip_subdir = "dummy_archive_path"
+    zip_filename = "archive.zip"
+    """
+        filenames = ["dummy_url path1", "dummy_url path12" ]
+    """
```

Comment:
this docstring is wrongly placed, should be after the function definition and properly writing to depict what the function is doing

Reply:
i have also corrected placement of the docstrings; placed them inside the function
i checked other endpoints on the codebase and saw that it followed this endpoint route patern